src/YoutubeAtom.tsx

```diff
@@ -225,6 +231,20 @@ const videoDurationStyles = css`
     color: ${palette['news'][500]};
 `;
 
+type YoutubeCallback = (
+    e: YT.PlayerEvent & YoutubeStateChangeEventType,
+) => void;
+
+// youtube-player doesn't have a type definition, do we have to create our own based on https://github.com/gajus/youtube-player
+type YoutubePlayerType = {
+    on: (state: string, callback: YoutubeCallback) => YoutubeCallback;
+    off: (callback: YoutubeCallback) => void;
+    loadVideoById: (videoId: string) => void;
+    playVideo: () => void;
+    getCurrentTime: () => number;
+    getDuration: () => number;
+};
```

Comment:
Can we move this up the page so that it is defined before it's used (more for readability reasons than anything)